package.json

```diff
@@ -25,13 +25,17 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "babel src/ -d lib/",
+    "deploy": "react-scripts build",
```

Comment:
Added deploy script since `react-scripts build` (which was originally the `npm run build` script) actually makes a minimal version of our front-end that can be used to serve out.

Reply:
Cool